eras/conway/impl/cddl-files/conway-merged.cddl

```diff
@@ -0,0 +1,506 @@
+; This file was auto-generated by cuddle-cddl. Please do not modify it directly!
+$hash28 = bytes .size 28
+
+$hash32 = bytes .size 32
+
+$kes_signature = bytes .size 448
+
+$kes_vkey = bytes .size 32
+
+$signature = bytes .size 64
+
+$vkey = bytes .size 32
+
+$vrf_cert = [bytes, bytes .size 80]
+
+$vrf_vkey = bytes .size 32
+
+addr_keyhash = $hash28
+
+address = h'001000000000000000000000000000000000000000000000000000000011000000000000000000000000000000000000000000000000000000'
+           / h'102000000000000000000000000000000000000000000000000000000022000000000000000000000000000000000000000000000000000000'
+           / h'203000000000000000000000000000000000000000000000000000000033000000000000000000000000000000000000000000000000000000'
+           / h'304000000000000000000000000000000000000000000000000000000044000000000000000000000000000000000000000000000000000000'
+           / h'405000000000000000000000000000000000000000000000000000000087680203'
+           / h'506000000000000000000000000000000000000000000000000000000087680203'
+           / h'6070000000000000000000000000000000000000000000000000000000'
+           / h'7080000000000000000000000000000000000000000000000000000000'
+
+anchor = [anchor_url : url, anchor_data_hash : $hash32]
+
+asset_name = bytes .size (0 .. 32)
+
+auth_committee_hot_cert = (14
+                           , committee_cold_credential
+                           , committee_hot_credential)
```

Comment:
Can we improve pretty printer a bit to format it a bit nicer? Feel free to just create tickets on cuddle for later for formatting improvements
```suggestion
auth_committee_hot_cert = ( 14
                          , committee_cold_credential
                          , committee_hot_credential
                          )
```

Reply:
Yep, the pretty printer right now is pretty basic, can definitely be improved!

Reply:
https://github.com/input-output-hk/cuddle/issues/34